lib/core/versions/latest/VerificationRelationship.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Sidetree public key verificationRelationship.
+ */
+enum VerificationRelationship {
```

Comment:
Please revert and stash this part of the change from the PR because:

1. They are unrelated to the property casing change.
1. This has further impact to downstream clients/services which they are not aware of and did not agree to
1. The current design of separation of patch actions and DID document output is intentional.

Adding @csuwildcat and @troyronda since this would require specification changes on both the core Sidetree protocol and the REST API, and they are editors in these spec. I am asking this part to be reverted so the casing change can be merged without being stalled by discussions on this.

Reply:
Downstream clients should pin to the latest stable version, so that we can implement many breaking changes all at once, and let them catch up in chunks.

Everyone who uses Verifiable Credentials and Encrypted Data Vaults needs these verification relationships...

https://github.com/transmute-industries/sidetree.js/blob/master/packages/common/src/enums/PublicKeyPurpose.ts#L23

https://github.com/trustbloc/sidetree-core-go/blob/28fabda71792f0bf59735ddbc9fa38bebcca1a32/pkg/dochandler/transformer/didtransformer/transformer.go#L130

https://w3c.github.io/did-core/#verification-relationships

Rather than push another set of breaking changes, I would prefer to simple correct the string enum "auth" to "authentication" and "general" to "verificationMethod" to align with did core.

I also added string enum values which match the exact bucket names in DID Core for the other missing relationships.

I'm afraid if we don't add support for these (and fix the internal naming confusion) in this PR, subsequent breaking changes will no longer be possible.

Better to get it done now.

Reply:
I disagree. But since this is contentious and technically unrelated to the casing change, please stash this part of the change for a separate discussion.

Reply:
Seems like the consensus on the call was to take these changes, and align with the W3C DID Core Spec.

Reply:
It's my preference that we get all the breaking changes in to adhere to the style guide now and then we don't need to change in the future. In this case, that would mean I'd prefer to keep `verificationRelationship`, but pluralize it to be `verificationRelationships`. See above comment for rationale.